src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -539,13 +534,13 @@ export class AppComponent implements OnInit, AfterViewChecked {
     }
 
     /**
-     * Format all text to replace /n with spaces
+     * Format all text to replace all new lines with spaces
      */
     textFormatter(row: number, cell: any, value: string, columnDef: any, dataContext: any): string {
         let valueToDisplay = value;
         let cellClasses = 'grid-cell-value-container';
         if (value) {
-            valueToDisplay = value.replace(/\n/g, ' ');
+            valueToDisplay = value.replace(/(\r\n|\n|\r)/g, ' ');
```

Comment:
Yeah, it fixed the bug still. I was slightly confused also. UDF definitions seem to be formatted differently from other output.